### PR TITLE
Naming convention for AngularJS

### PR DESCRIPTION
**Pull request by ryosuzuki**
Related issue: #238

### CODE REVIEW

File: staticfiles/js/tasks/controllers/task.list.controller.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+(function() {
+  'use strict';
+
+  angular.module('crowdsource.tasks.controllers', ['smart-table'])
+    .factory('Task', Task);
+  Task.$inject = ['$http'];
+
+  angular.controller('TaskNewController', TaskController);
```
Comment (nistala) [inline staticfiles/js/tasks/controllers/task.list.controller.js]: should this be task list?

**Comment by nistala** [comment]
Any reason in particular for this naming convention? With angular we won't need separate controllers such as search unless we want a new page for them.

**Comment by dmorina** [comment]
best practice and easiest to maintain is to group things by functionality, currently we are doing well so I don't think we should change it, let's revisit this later after  a month or so.. this doesnt mean that people can use random structures though, it's very simple group things that go together, and if the code is being shared by multiple pages then put it in a shared container

**Comment by neilthemathguy** [comment]
+1

**Comment by ryosuzuki** [comment]
OK, I understand.
We should focus on the existing issues

**Pull request closed without merging**